Create ui-c8y-1021-30-1-alias-cannot-be-empty-7817-graft-release-cd-7882.md (#2781)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-1021-30-1-alias-cannot-be-empty-7817-graft-release-cd-7882.md

---
date: ""

> title: The login alias cannot be an empty string.

Reviewer: ```suggestion
title: Login alias can no longer be an empty string
```

Reviewer: @mmed-c8y What is the fix here?

> title: alias cannot be empty (#7817) [GRAFT][release/cd] (#7882)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Fix error when creating an empty alias for a device
```

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-0UgqXH1Ys
    label: Administration
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-61921
version: 1021.30.1
---

> alias cannot be empty (#7817) [GRAFT][release/cd] (#7882)

Author: The Claude AI suggested to use the following body:
```suggestion
In some cases, it was possible to create an empty alias for a device, which could lead to unexpected behavior in the UI and API. With this change, an error is now returned when attempting to create an empty alias for a device. This prevents invalid device aliases from being created, ensuring more consistent and reliable device management for users.
```

> The login alias cannot be an empty string, a string consisting only of whitespace, null, or undefined. If a user does not have a login alias, this field should not be included in the user object.

Reviewer: ```suggestion
The login alias cannot be an empty string, that is, a string consisting only of whitespace, null, or undefined. If a user does not have a login alias, this field must not be included in the user object.
```

> The login alias cannot be an empty string, that is, a string consisting only of whitespace, null, or undefined. If a user does not have a login alias, this field must not be included in the user object.

Reviewer: ```suggestion
The login alias can no longer be an empty string, that is, a string consisting only of whitespace, null, or undefined. If no login alias is provided for the user, this field must not be included in the user object.
```
